Signal/src/call/Speakerbox/CallKitCallUIAdaptee.swift

```diff
@@ -234,6 +242,19 @@ final class CallKitCallUIAdaptee: NSObject, CallUIAdaptee, CXProviderDelegate {
         _ = self.callService.handleOutgoingCall(call)
         action.fulfill()
         self.provider.reportOutgoingCall(with: call.localId, startedConnectingAt: nil)
+
+        ensureCallName(call:call)
```

Comment:
Inlined it.  

> we're reporting the call, not just munging the name.

You comment makes me unsure that I understand the CallKit API.  The intent her is to "set" (not munge) the name.  When "CallKit privacy" is enabled, we create new incoming and outgoing calls using a "generic" handle, not a phone number, so we have to set the `localizedCallerName` once the call is created.  Otherwise, CallKit presents the call to the user using the "generic" UUID.  So this `CXCallUpdate` is used to overwrite the UUID with the "Signal User" copy.  

> we're reporting the call

My understanding is that `CXCallUpdate` and `provider.reportCall` are how you update properties of a call.  Your comment makes it sounds like there's more to it.  What am I missing?